LinGui: tweak About dialog so Close button works

foutrelis opened the pull request:
Clicking on "Close" emits a "response" signal which was ignored. Make
the button work by calling gtk_dialog_run(), as it's done for similar
dialogs in callbacks.c.

Fixes https://github.com/HandBrake/HandBrake/issues/2405

_Tested on Xfce 4.14 with GTK+ 3.24.20._

jstebbins commented on gtk/src/callbacks.c:
```diff
@@ -4532,17 +4532,18 @@ ghb_browse_uri(signal_user_data_t *ud, const gchar *uri)
 about_action_cb(GSimpleAction *action, GVariant *param, signal_user_data_t *ud)
 {
-    GtkWidget *widget = GHB_WIDGET (ud->builder, "hb_about");
+    GtkWidget *dialog = GHB_WIDGET (ud->builder, "hb_about");
```
First of all, thanks.  Solves the problem without affecting newer gtk versions where the dialog button does not appear.

But could you please revert the change to the `widget` variable name.  It's not necessary and makes a 2 line patch into a patch that affects pretty much every line in the function.

foutrelis replied:
Sorry about that; diff does look much cleaner now. :)

foutrelis committed:
LinGui: tweak About dialog so Close button works

Clicking on "Close" emits a "response" signal which was ignored. Make
the button work by calling gtk_dialog_run(), as it's done for similar
dialogs in callbacks.c.

Fixes https://github.com/HandBrake/HandBrake/issues/2405
